_data/tutorials.yaml

```
@@ -484,3 +483,24 @@ masking-data:
     ruby: disabled
     scala: disabled
     swift: disabled
+
+anomaly-detection:
+  title: "Detecting abnormal events"
+  meta-description: "Detecting abnormal events"
+  slug: "/anomaly-detection"
+  question: "How do I find abnormal events?"
+  introduction: "Consider a topic with events containing information about financial transactions and a table with known suspicious names. A common pattern for fraudsters is to disguise transactions under the name of a popular company, the idea being that the chances of them being recognized is very low. For example, transactions labeled Verizon, Citibank, USPS, etc., are likely to look similar and blend in.
```

Feel free to wordsmith...

```suggestion
  introduction: "A common pattern for fraudsters is to disguise transactions under the name of a popular company, the idea being that the chances of them being recognized is very low. For example, transactions labeled Verizon, Citibank, USPS, etc., are likely to look similar and blend in.  This tutorial shows you how to identify this pattern of behavior by detecting 'abnormal' transactions that occur within a window of time
```